tests/unit/test_session.py

```diff
@@ -3102,7 +3109,7 @@ def test_endpoint_from_production_variants_with_sagemaker_config_injection(
     sagemaker_session.sagemaker_client.create_endpoint.assert_called_with(
         EndpointConfigName="some-endpoint",
         EndpointName="some-endpoint",
-        Tags=expected_tags,  # from config
+        Tags=[],
```

Comment:
Why this can't be matched with `expected_tags` above from SAGEMAKER_CONFIG_ENDPOINT_CONFIG?

Reply:
Added a new UT with the combination in the next commit :)